src/pages/WizzardPage2/WizzardPage2.js

```diff
@@ -0,0 +1,307 @@
+import { useEffect, useState, useContext } from "react";
+
+import classes from "./WizzardPage2.module.scss";
+import Input from "../../components/Wizzard/Input/Input";
+import Select from "../../components/Wizzard/Select/Select";
+import useInput from "../../hooks/useInput";
+import WizzardFormLayout from "../../layout/WizzardFormLayout/WizzardFormLayout";
+import WizzardButtons from "../../components/Wizzard/WizzardButtons/WizzardButtons";
+import { Contex } from "../../store/Store";
+
+const WizzardPage2 = (props) => {
+  const { onForm2Submit, setStep } = props;
+  const ctx = useContext(Contex);
+  const wizardData = ctx.state.applicationInfo;
+
+  const [state, setState] = useState({
+    email: wizardData.email,
+    address1: wizardData.address1,
+    address2: wizardData.address2,
+    country: wizardData.country,
+    city: wizardData.city,
+    postalCode: wizardData.postalCode,
+    howManyYearsLived: wizardData.howManyYearsLived,
+  });
+
+  console.log(state.country);
+
+  const [formIsValid, setFormIsValid] = useState(false);
+
+  const emailInputHandler = (e) => {
+    setState((prev) => ({ ...prev, email: e.target.value }));
+  };
+  const {
+    onChangeFunc: emailChangeFunc,
+    onBlurFunc: emailBlurFunc,
+    isValid: emailIsValid,
+    isInvalid: emailInvalid,
+  } = useInput(
+    (val) => {
+      return val.trim() !== "" && val.includes("@");
+    },
+    emailInputHandler,
+    state.email
+  );
+
+  const address1InputHandler = (e) => {
+    setState((prev) => ({ ...prev, address1: e.target.value }));
+  };
+  const {
+    onChangeFunc: address1ChangeFunc,
+    onBlurFunc: address1BlurFunc,
+    isValid: address1IsValid,
+    isInvalid: address1Invalid,
+  } = useInput(
+    (val) => {
+      return val.length !== 0;
```

Comment:
Better way of writing this would be return !val.length. Use falsy and truthy values for 0, "",... Don't forget that an empty array and empty object is a truthy value, so you should always check for the length of an array.